spec/hyrax/transactions/steps/set_uploaded_date_unless_present_spec.rb

```diff
@@ -50,6 +50,18 @@
           .to xmas
       end
 
+      context 'when an uploaded date exists' do
+        let(:xmas_past) { DateTime.parse('2009-12-25 11:30').iso8601 }
+
+        before { change_set.date_uploaded = xmas_past }
+
+        it 'sets the uploaded date to the modified date' do
```

Comment:
I don't think this description matches the tested behavior.

Consider `it "does not overwrite the existing uploaded date"`